test/parallel/test-repl.js

```diff
@@ -180,6 +180,13 @@ function error_test() {
       expect: prompt_multiline },
     { client: client_unix, send: '})()',
       expect: '1' },
+    // Multiline function call
+    { client: client_unix, send: 'function f(){}; f(f(1,',
+      expect: prompt_multiline },
+    { client: client_unix, send: '2)',
+      expect: prompt_multiline },
+    { client: client_unix, send: ')',
+      expect: 'undefined' },
```

Comment:
I think `prompt_unix` should immediately follow the `undefined`. Can you please test and include that as well?